_includes/headtags.html

```diff
@@ -97,5 +100,13 @@
 {% endif %}
 
 {% if page.layout == "post" and page.comments != false %}
-<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+<script>
+    var recaptchaOnload = _ => {
+        grecaptcha.render('captcha__comment__form', {
+          'sitekey' : '{{ site.reCaptcha.siteKey }}',
+          'theme' : window.THEME ? Window.THEME : "light"
```

Comment:
I think it is `window.THEME`, not `Window.THEME` (after the `?`). Found while testing

Reply:
done